core/docker/bin/run-trino

```diff
@@ -7,4 +7,34 @@ if ! grep -s -q 'node.id' /etc/trino/node.properties; then
     NODE_ID="-Dnode.id=${HOSTNAME}"
 fi
 
+base=/usr/lib/trino
+if [ -n "${TRINO_ENABLE_PLUGINS:-}" ] && [ -n "${TRINO_DISABLE_PLUGINS:-}" ]; then
+    echo >&2 "Cannot set both \$TRINO_DISABLE_PLUGINS and \$TRINO_ENABLE_PLUGINS"
+    exit 1
+elif [ -n "${TRINO_ENABLE_PLUGINS:-}" ]; then
+    if [ -n "${TRINO_DISABLE_PLUGINS:-}" ]; then
+        echo >&2 "Cannot set both \$TRINO_DISABLE_PLUGINS and \$TRINO_ENABLE_PLUGINS"
+        exit 1
+    fi
+    mv "$base/plugin" "$base/plugin.disabled"
+    mv "/etc/trino/catalog" "/etc/trino/catalog.disabled"
+    mkdir -p "$base/plugin" /etc/trino/catalog
+    IFS=, read -ra names <<< "$TRINO_ENABLE_PLUGINS"
+    for name in "${names[@]}"; do
+        mv "$base/plugin.disabled/$name" "$base/plugin/$name"
+        if [ -f "/etc/trino/catalog.disabled/$name.properties" ]; then
```

Comment:
plugin name != catalog name

did you mean TRINO_ENABLED_CATALOGS env var?

Reply:
Whoops, you're right. I added this because there are default catalogs that will cause the server to fail at startup if the connectors/plugins they use are disabled. I want to keep this simple so I'd like to avoid adding even more env vars. Would grepping for `connector.name=$name` be ok instead?

Reply:
That's another reason to only do this at build time - to avoid adding new logic to parse catalog configs outside of Java.